idact/detail/add_cluster_app/main.py

```diff
@@ -47,7 +44,10 @@
               type=str,
               help="In order for idact to find and execute the proper "
                    "binaries, they must be specified as a list of "
-                   "Bash script lines.")
+                   "Bash script lines. "
+                   "NOTE: RuntimeError: Retried and failed: config.retries"
```

Comment:
This information is misleading, the error is not caused by a lack of 'actions file'. It happens when we deploy notebook on a cluster without python-intel 3.6.2. I know it can be solved by specifying a 'module load' in actions file but it would be better to inform a user about this after unsuccessful deployment

Reply:
removed